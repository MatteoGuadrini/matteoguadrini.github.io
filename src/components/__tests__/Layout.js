import React from "react"
import renderer from "react-test-renderer"
import Layout from "../layout"
describe("Header", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<Layout><div/></Layout>)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
