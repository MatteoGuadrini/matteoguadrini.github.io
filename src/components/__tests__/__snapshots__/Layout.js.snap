// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<div
  className="yui-t7"
  id="doc2"
>
  <div
    id="inner"
  >
    <div />
  </div>
  <footer>
    Â© 
    2019
    , Built with
     
    <a
      href="https://www.gatsbyjs.org"
    >
      Gatsby
    </a>
  </footer>
</div>
`;
